includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -119,11 +119,13 @@ public function get_stripe_supported_countries() {
 	 * @return bool
 	 */
 	public function are_keys_set() {
-		if ( empty( $this->secret_key ) || empty( $this->publishable_key ) ) {
-			return false;
+		if ( $this->testmode ) {
```

Comment:
Since this isn't DRY, and I assume making it so would be burdensome, should we add a comment here (and there) directing maintainers to take care of the other code too?

Reply:
addressed in 6f926b7a6239ec67cc6d3b6038f2d9dff714dc99